react-project/src/Components/Item I/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
```

Comment:
Add semicolon after every statement. You can install Prettier vscode extension that does this for you